### ISSUE TITLE
WebDriverFacade.isEnabled() not always consulted

### ISSUE THREAD

**Issue by csabakoncz** [serenity-bdd/serenity-core#450]
Some of the `WebDriverFacade` methods check the `isEnabled()` state and perform no action or return a stub (e.g. `get()`, `findElement()`, etc).

However, some methods do not perform this check which can lead to unexpected exceptions. E.g. in executeScript() 

```
public Object executeScript(String script, Object... parameters) {
     return ((JavascriptExecutor) getProxiedDriver()).executeScript(script, parameters);
}
```

the unchecked cast to JavaScriptExecutor can fail:
`java.lang.ClassCastException: net.thucydides.core.webdriver.stubs.WebDriverStub cannot be cast to org.openqa.selenium.JavascriptExecutor`

The problem does not occur under "normal" circumstances, however, it often appears when I try to use multiple JBehave threads.

**Comment by wakaleo** [comment]
Thanks for spotting this, it will be fixed in the next release.

**Comment by wakaleo** [comment]
Fixed in 1.1.37-rc.5
